src/components/QuantitySelector.js

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import { FaPlus, FaMinus } from "react-icons/fa";
+
+const QuantityButton = styled.button`
+    background: ${(p) => p.theme.colors.background};
+    color: ${(p) => p.theme.colors.text};
+    border: none;
+`
+
+const handleDecrease = (value, setValue) => {
+    if (value === 0) return; 
+    setValue(value - 1)
+}
+
+const handleIncrease = (value, setValue) => {
+    setValue(value + 1)
+}
+
+const QuantitySelectorInput = styled.input`
+    width: 2ch;
+    height: 12px;
+    font-size: 14px;
+    font-family: ${(p) => p.theme.fonts};
+    font-weight: ${(p) => p.theme.fontWeight};
+    color: ${(p) => p.theme.colors.text};
+    cursor: not-allowed;
+    margin: 3px;
+    padding: 10px;
+    border: none;
+    border-radius: 3px;
+    text-align: center; 
+    
+    :placeholder{
+        color: ${(p) => p.placeholderColor ?? "black"}
```

Comment:
are we sure we want to be using `??` here? What is the behaviour we expect if `p.placeholderColor` is an empty string? Should we default to black in that case?

Reply:
I think it's fine cause it's not coming from user, we prob wouldn't see just `""`